tests/_async/test_search.py

```diff
@@ -479,22 +479,26 @@ def test_complex_example() -> None:
 def test_reverse() -> None:
     d = {
         "query": {
-            "filtered": {
```

Comment:
The `filtered` query was deprecated around ES 2.x and does not exist anymore in the schema. I replaced it with `bool`.